WordPress/gutenberg
# FontSizePicker: Fix header order in RTL languages

## What?
Addresses a bug noticed by @ciampo in https://github.com/WordPress/gutenberg/pull/44483#pullrequestreview-1123284658 where the _Size_ label and hint appear in the wrong order in RTL languages.

## Why?

## How?
Setting `display: inline-block` makes the browser re-order the elements automatically because `direction: rtl` will apply to all non `inline` elements. Setting `margin-right` instead of `margin-left` when the language is RTL ensures that the margin goes on the correct side of the hint.

## Testing Instructions
Best to test this in the real app as the RTL setting in Storybook doesn't affect the result of `wp.i18n.isRTL()`.

1. Set your language to e.g. Hebrew or Arabic.
2. Navigate to Appearance → Editor → Global Styles → Typography → Paragraph.
3. The hint should appear to the left of the _Size_ label.

## Screenshots or screencast 
| Before | After |
| - | - |
| <img width="303" alt="Screen Shot 2022-09-30 at 11 34 58" src="https://user-images.githubusercontent.com/612155/193171701-468d9488-5824-43b9-9b81-79771b434d5f.png"> | <img width="302" alt="Screen Shot 2022-09-30 at 11 35 06" src="https://user-images.githubusercontent.com/612155/193171707-0ff93539-f251-4cbf-878d-123595233b29.png"> |
